manim/mobject/svg/svg_mobject.py

```diff
@@ -510,17 +510,15 @@ def handle_commands(self) -> None:
         all_points: list[np.ndarray] = []
         last_move = None
         curve_start = None
+        last_true_move = None
 
-        # These lambdas behave the same as similar functions in
-        # vectorized_mobject, except they add to a list of points instead
-        # of updating this Mobject's numpy array of points. This way,
-        # we don't observe O(n^2) behavior for complex paths due to
-        # numpy's need to re-allocate memory on every append.
-        def move_pen(pt):
-            nonlocal last_move, curve_start
+        def move_pen(pt, true_move=False):
```

Comment:
```suggestion
        def move_pen(pt, *, true_move: bool = False):
```
For code readability, it might be better to make it a keyword only parameter.